### ISSUE TITLE
Add configuration property to allow switch off forcing prompt consent for offline access

### ISSUE THREAD

**Issue by yuriyz** [GluuFederation/oxAuth#1496]
# Describe the issue

Connect spec
```
When offline access is requested, a prompt parameter value of consent MUST be used unless other conditions for processing the request permitting offline access to the requested resources are in place. The OP MUST always obtain consent to returning a Refresh Token that enables offline access to the requested resources. A previously saved user consent is not always sufficient to grant offline access.
```
We are forcing it now
```
AuthorizeRestWebServiceImpl.java:333) - Removed offline_access. Can't find prompt=consent. Consent is required for offline_access.
```

However there can be cases when we wish to switch off this restriction.

**Comment by yuriyz** [comment]
We will not introduce new property, instead `pre-authorization flag` will take higher priority. If it's true then we will ignore `consent MUST`.

**Comment by yuriyz** [comment]
Corrected in 4.2.2 and jans.
